pkg/dtos/user.go

```diff
@@ -14,7 +14,7 @@ type User struct {
 	DisplayName string   `json:"displayName"`
 	Password    string   `json:"password,omitempty" validate:"required,edgex-dto-none-empty-string,edgex-dto-password"`
 	Description string   `json:"description"`
-	Roles       []string `json:"roles,omitempty"`
+	Roles       []string `json:"roles,omitempty" validate:"unique"`
```

Comment:
Please could you add a unit test in https://github.com/IOTechSystems/go-mod-central-ext/blob/0f42cbd87d3918f3ec4da8b3d7c62ff8cad336e4/pkg/dtos/requests/user_test.go#L52 to validate the duplicate role case? Thanks.